.github/workflows/ci.yml

```
@@ -67,7 +67,7 @@ jobs:
       run: eipv EIPS/ --ignore=title_max_length,missing_discussions_to --skip=eip-20-token-standard.md
 
   eipw-validator:
-    name: EIPW Validator
+    name: EIP Walidator
```

```suggestion
    name: EIPU Ualidator
```
If it's a w (double-u) then I believe that it should really be corrected to this.

In all seriousness, I wasn't aware that you had agreed with https://github.com/ethereum/EIPs/issues/5315. Good to know :)

---

I haven't!  I really do want to address the problem of EIPW being neither/both an acronym and a word at the same time.  Either it should be named `eipw` (not an acronym, the letters don't mean anything), or it should be `EIP W???` where the W is the first letter of some word.  I don't actually care what that word is, and it can be a backronym.  I just don't think it should be `EIPW Validator` because that implies that it is something that validates EIPWs, and those aren't a thing.

---

```suggestion
    name: WIP Validator
```
How about this? It does validate WIP EIPs :)